eras/byron/ledger/executable-spec/test/Test/Byron/Spec/Ledger/AbstractSize/Properties.hs

```diff
@@ -153,5 +153,5 @@ testTxHasTypeReps =
     "Test HasTypeReps instances"
     [ testCase "AbstractSize - example - TxIn" exampleTypeRepsTxIn,
       testCase "AbstractSize - example - Tx" exampleTypeRepsTx,
-      testProperty "AbstractSize and HasTypeReps - Tx*" propTxAbstractSize
+      testPropertyNamed "AbstractSize and HasTypeReps - Tx*" "propTxAbstractSize" propTxAbstractSize
```

Comment:
Looks like you are missing `OverloadedString` extension. However, I'd rather leave byron codebase alone, unless a change is really necessary

Reply:
Reverted.